### PR TITLE
add rule for Eyeonx.ch

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

File: src/chrome/content/rules/eyeonx.ch.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!--
+		Cert expired:
+		-	event.eyeonx.ch
```
Comment (J0WI) [inline src/chrome/content/rules/eyeonx.ch.xml]: See also `converter.eyeonx.ch`

Comment (numismatika) [inline src/chrome/content/rules/eyeonx.ch.xml]: every subdomain of eyeonx.ch is answered for by a wildcard dns on deliberate typos.
What about exactly this domain?

**Comment by J0WI** [comment]
Referring to Travis there are some more expired certs.

**Comment by numismatika** [comment]
@J0WI could you fix the label now that checks are green?

**Pull request merged**